docs/17-porting-tips.md

```diff
@@ -9,7 +9,7 @@ you might have added to your `rc.lua`.
 This avoids most of the possible errors due to missing important changes.
 
 If you still wish to ignore this advice, first read the
-<a href="89-NEWS.md.html#v4">NEWS</a> section about the breaking changes. This
+[NEWS](89-NEWS.md) section about the breaking changes. This
```

Comment:
I'm not sure about this one. The `#v4` was added there on purpose.

Does your ikiwiki generate a different file name than `89-NEWS.md` or why does this link to be fixed?